### PR TITLE
added info-ptbr

### PR DESCRIPTION
**Pull request by mezysinc**

### CODE REVIEW

File: app/src/main/assets/info-ptbr.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+</div>
+<div>Em particular, agradecimentos a:</div>
+<div>
+    <ul>
+        <li><a href="http://martincesare.com.ar/">Martín Césare</a> por criar toda a gráfica de software.
+        </li>
+        <li><a href="http://www.canonical.com/">Canonical</a> por criar este maravilhoso distro GNU/Linux.
+        </li>
+        <li><b>Mazoyer</b> pela tradução em francês.</li>
+        <li><b>Monino</b> pela tradução em espanhol.</li>
```
Comment (leinardi) [inline app/src/main/assets/info-ptbr.html:62]: If you want, feel free to add yourself to this list :)

Comment (mezysinc) [reply app/src/main/assets/info-ptbr.html:62]: doesn't really matter ;)

Comment (mezysinc) [reply app/src/main/assets/info-ptbr.html:62]: It's there now :)

**Review by mezysinc — COMMENTED** [review]
Looks good

**Comment by leinardi** [comment]
Thanks!

**Pull request merged**